### DISCUSSION TITLE
Biggest feature

### DISCUSSION THREAD

**Discussion by sgamesdev** [Leo-Corporation/Gavilya#276] [Ideas]
The biggest feature you can implement in this project would be an fps meter it in game overlay

**Comment by lpeyr** [comment]
This is a great idea, but it is difficult to implement. I will try anyway, but it may take some time.

**Reply by sgamesdev** [reply]
I have the code for the fps meter , but not for the ingame overlay

**Reply by sgamesdev** [reply]
https://stackoverflow.com/questions/18340347/monitoring-the-fps-of-a-direct-x-application

**Reply by lpeyr** [reply]
Thanks! I'll try to implement it in a future update. I will make the overlay myself.
